### ISSUE TITLE
Can't upgrade Erekir cores in sandbox

### ISSUE THREAD

**Issue by overdramaticpanromantic** [Anuken/Mindustry#9027]
### Platforms

Mac

### Build

Newer 146

### Issue

When attempting to upgrade a Serpulo core to an Erekir core, it does not let you, stating "Core Base Required."

### Steps to reproduce

1. Place a Serpulo core in sandbox by any means.
2. Select any Erekir core. For example, an Acropolis if you're upgrading a Nucleus.
3. Attempt to upgrade the core, and witness the bug.

### Mods used

None.

### Save file

[cant_upgrade_cores.msav.zip](https://github.com/Anuken/Mindustry/files/12525175/cant_upgrade_cores.msav.zip)
Notify me if the file failed to upload.

### (Crash) logs

No crash.

### Submission

- [X] I have updated to the latest release (https://github.com/Anuken/Mindustry/releases) to make sure my issue has not been fixed.
- [X] I have searched the closed and open issues to make sure that this problem has not already been reported.

**Comment by overdramaticpanromantic** [comment]
You were able to do this before 146, though. It doesn't appear to be unintended functionality, but I may be wrong. Also worth noting this affects modded cores, so if a mod has multiple core trees, you can't use both.

**Comment by Anuken** [comment]
> You were able to do this before 146, though. 

Yes, it was a bug that was fixed. See: [this Discord discussion.](https://discord.com/channels/391020510269669376/396416151032299521/1134955590343475311)

> Also worth noting this affects modded cores, so if a mod has multiple core trees, you can't use both.

It doesn't affect modded cores unless they (1) require core zone tiles, and (2) don't have core zone tiles present on that map.

**Comment by overdramaticpanromantic** [comment]
Ah.
